changelogs/unreleased/gh-7746-fiber-channel-graceful-close.md

```diff
@@ -0,0 +1,5 @@
+## feature/fiber
+
+* Fiber channel now has a compatibility option in tarantool.compat, that
+  provides transition from force channel `close()` to graceful `close()`, i.e.
+  closing channel for writing while existing messages can still be extracted.
```

Comment:
Please, leave an issue number `<...> (gh-xxxx).` at the end.

Reply:
Done.